### PR TITLE
refactor images, change heading colors on scroll

### PR DESCRIPTION
**Pull request by chloehwang**
This PR worked on images and font colors:

- refactored how we pull in images using the Gatsby way of importing them
- added info for the rest of the sections in `index.json`
- added corresponding colors for the rest of the sections
- added scss to change the color of headings/subheadings/buttons on scroll

### CODE REVIEW

File: packages/microcosm-www-next/src/stylesheets/structure/_section.scss [hunk 1]
```diff
@@ -224,6 +249,10 @@ $section-top-padding-desktop: $nav-height + 110px;
   &:last-child {
     @include breakpoint(large-desktop-up) {
       margin-bottom: 60vh;
+
+      &::after {
+        content: ''
+      }
     }
   }
```
Comment (chloehwang) [inline packages/microcosm-www-next/src/stylesheets/structure/_section.scss]: below are just placeholder images that I'll swap out in another PR

**Comment by chloehwang** [comment]
@leobauza the CI website test is failing again but I think that's okay according to @nhunzaker?

**Comment by nhunzaker** [comment]
Pretty strange. Let's not worry about it. I wish I knew what was up tho :(

**Comment by chloehwang** [comment]
@nhunzaker yeah,  it sometimes works and sometimes doesn't so I wonder if it's a CI thing? anyways, will keep moving

File: packages/microcosm-www-next/src/components/graphic.js [hunk 2]
```diff
@@ -1,13 +1,18 @@
...
     className="section__graphic__figure"
     data-module="ObserveGraphic"
     data-section={section}
   >
-    <img src={graphicUrl} alt="TODO" />
+    <img
+      src={
+        microcosmView ? `/${section}-microcosm.png` : `/${section}-browser.png`
+      }
+      alt="TODO"
```
Comment (leobauza) [inline packages/microcosm-www-next/src/components/graphic.js]: ?

Comment (chloehwang) [reply packages/microcosm-www-next/src/components/graphic.js]: I'll replace all the `alt="TODO"` at the end

**Pull request merged**